packages/dds/tree/src/test/simple-tree/treeApi.spec.ts

```diff
@@ -768,5 +768,147 @@ describe("treeNodeApi", () => {
 			assert.equal(nodeChanged, true, "'nodeChanged' should have fired");
 			assert.equal(treeChanged, true, "'treeChanged' should have fired");
 		});
+
+		it(`'nodeChanged' includes the names of changed properties (objectNode)`, () => {
+			const sb = new SchemaFactory("test");
+			class treeSchema extends sb.object("root", {
```

Comment:
class names, including schema should generally be capitalized.

types of nodes (when specified via schema) generally shouldn't end with "schema" (`const foo: Foo = new Foo({})` is more natural than (`const foo: FooSchema = new FooSchema({})` since Foo is refering to the type of the node, not the type of the schema (the type of the schema is `typeof Foo`)

```suggestion
			class TestNode extends sb.object("root", {
```